test/Riok.Mapperly.Tests/Mapping/EnumExplicitMapTest.cs

```diff
@@ -63,7 +64,9 @@ public void EnumByNameWithExplicitValueDuplicatedTarget()
                 };
                 """
             )
-            .HaveDiagnostic(new(DiagnosticDescriptors.TargetEnumValueNotMapped, "Enum member D (3) on E1 not found on source enum E2"));
+            .HaveDiagnostic(DiagnosticDescriptors.TargetEnumValueNotMapped, "Enum member D (3) on E1 not found on source enum E2")
```

Comment:
Could be simplified to `.HaveDiagnostic(DiagnosticDescriptors.TargetEnumValueNotMapped, "Enum member D (3) on E1 not found on source enum E2", "Enum member F (6) on E1 not found on source enum E2")`

Reply:
Fixed.